tutorials/OLD_grafana/grafana-timescalecloud.md

```diff
@@ -45,32 +45,37 @@ and then configure Timescale as the data source in Grafana.
 To configure TimescaleDB service on Timescale with your Grafana
 installation, log in to Grafana and proceed to step 5 in this procedure.
 
-1.  Sign in to your Managed Service for Timescale account, and click the name of
+1.  Sign in to your Timescale account, and click the name of
     your new Grafana service.
 1.  On the service details page, take a note of the `User` and `Password` field for
     your service.
-1.  Click the link in the `Service URI` field to open Grafana.
-1.  Log in to Grafana with your service credentials.
+1.  Navigate over to Grafana and log in with your service credentials.
 1.  Navigate to `Configuration` → `Data sources`. The data sources page lists
-    previously configured data sources for the Grafana instance.
-1.  Click `Add data source` to see a list of all supported data sources.
-1.  Type `PostgreSQL` in the search field and click `Select`.
-1.  Configure the data source:
+    supported data sources previously configured for the Grafana instance.
+1.  Click `Add data source`.
+1. Type `PostgreSQL` in the search field and click `Select`. The choose a method of
+    connection.
+1.  To configure the data source:
```

Comment:
Configure the data source


This is to be consistent with the way the other steps are written.